docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
       - SSAS_PUBLIC_URL=${SSAS_PUBLIC_URL}
       - SSAS_TIMEOUT_MS=2000
       - BCDA_ENABLE_NEW_GROUP=true
-      - PRIORITY_ACO_IDS=A9990,A9991,A9992,A9993,A9994
+      - PRIORITY_ACO_REG_EX=([A-Z]99([0-8]|9[0-8])$$) # $$ signifies a literal $ in the string
```

Comment:
It is not clear to me why this regex contains `$$` but the regex env var defined in `bcda-ops` contains `$`.

Reply:
Hey Matt, if I keep it as a single `$` we get this Error `ERROR: Invalid interpolation format for "environment" option in service "api": "PRIORITY_ACO_REG_EX=([A-Z]99([0-8]|9[0-8])$)"`. At the VERY bottom of this doc https://github.com/compose-spec/compose-spec/blob/master/spec.md it talks about using double dollar signs when requiring a literal dollar sign.

Reply:
Ahh, cool. Thanks for the link to the reference doc.